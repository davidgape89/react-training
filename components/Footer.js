import React from 'react';

const Footer = (props) => (
  <div className="footer">
    netflixroulette
  </div>
);

export default Footer;