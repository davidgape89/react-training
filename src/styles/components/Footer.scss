@import '../variables';

.footer {
  background-color: $grey;
  color: $light-red;
  height: 55px;
  padding: 20px;
}