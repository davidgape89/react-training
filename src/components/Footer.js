import React from 'react';

import './Footer.scss';

const Footer = (props) => (
  <div className="footer">
    netflixroulette
  </div>
);

export {Footer};