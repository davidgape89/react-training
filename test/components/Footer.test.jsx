import React from 'react';
import {shallow} from 'enzyme';

import Footer from '../../components/Footer';

describe('Footer -', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  })

  it('renders correclty', () => {
    expect(wrapper).toMatchSnapshot();
  })
})