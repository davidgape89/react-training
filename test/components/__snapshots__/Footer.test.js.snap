// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer - renders correclty 1`] = `
<div
  className="footer"
>
  netflixroulette
</div>
`;
